src/test/kotlin/fi/metatavu/timebank/api/test/functional/impl/TimeEntriesTestBuilderResource.kt

```diff
@@ -46,6 +47,19 @@ class TimeEntriesTestBuilderResource(
         )
     }
 
+    /**
+     * delete time entry
```

Comment:
Typically the first letter of a comment is capitalized, there are a few cases where this could be updated